src/public/services/AdminFormService.ts

```diff
@@ -122,6 +122,15 @@ export const updateFormEndPage = async (
     .then(({ data }) => data)
 }
 
+export const createFormLogic = async (
+  formId: string,
+  createdLogic: LogicDto,
```

Comment:
nit: rename `createdLogic` to `createLogicBody` for clarity? (Since the logic has not actually been inserted, and is actually just a body parameter to the endpoint)

Reply:
thanks! edited